### PR TITLE
Use faker data in askama templates during dev

### PR DESCRIPTION
**Pull request by oeb25**
By adding a `$ {{lorem.limes}}` for example to ``{askama`...`}``, fake data is inserted during development.

Additionally all askama specific text (such as `{% for x in xs %}`) is not produced during development for cleaner pages.

### CODE REVIEW

File: frontend/src/components/Entity.astro [hunk 1]
```diff
@@ -38,11 +41,11 @@ export interface Props {
     {
       askama.if_("!entity.info.is_empty()", () => (
         <div class="mt-5 flex w-full flex-col px-4 text-sm">
-          <div class="grid border-separate border-spacing-x-4 border-spacing-y-1.5 grid-cols-[auto_1fr] gap-x-4">
+          <div class="border-spacing-x-4 border-spacing-y-1.5 grid border-separate grid-cols-[auto_1fr] gap-x-4">
             {askama.for_("(key, value) in entity.info", () => (
               <>
-                <div class="text-gray-400">{askama`key`}</div>
-                <div>{askama`value`}</div>
+                <div class="text-gray-500">{askama`key $ {{word.adjective}}`}</div>
```
Comment (oeb25) [inline frontend/src/components/Entity.astro:47]: `text-gray-400` is not accessible on while background.

Comment (mikkeldenker) [reply frontend/src/components/Entity.astro:47]: We should probably also have some more vertical spacing between the key-value pairs, but we can just change that later

File: frontend/src/components/Entity.astro [hunk 2]
```diff
@@ -14,19 +14,22 @@ export interface Props {
           <a href="https://en.wikipedia.org/wiki/{{ entity.title|urlencode }}">
             <img
               class="h-full w-full rounded-full"
-              src="/entity/image/{{ image|urlencode }}"
+              src={askama.fake(
+                "/entity/image/{{ image|urlencode }}",
```
Comment (mikkeldenker) [inline frontend/src/components/Entity.astro:18]: Just leaving a note here for the future. We should move this url prefix server-side

File: frontend/src/pages/search.astro [hunk 3]
```diff
@@ -124,7 +127,10 @@ import { askama } from "../askama";
                         <a href="{{ item.url|urlencode }}">
                           <img
                             class="h-full w-full rounded-full object-cover"
-                            src="/image/{{ val|urlencode }}"
+                            src={askama.fake(
+                              "/image/{{ val|urlencode }}",
```
Comment (mikkeldenker) [inline frontend/src/pages/search.astro:125]: Again, this url should also be server-side generated

**Review by mikkeldenker — APPROVED** [review]
I just left some comments for stuff we should probably fix in the future (nothing to do with this PR). The changes look very good!

**Pull request merged**